src/expo.ts

```diff
@@ -98,3 +98,17 @@ export function projectLink(project: ProjectInfo, channel?: string): string {
 
   return url.toString();
 }
+
+/**
+ * Create a deep link for the project in Expo.
+ */
+export function projectDeepLink(project: ProjectInfo, channel?: string): string {
+  assert(project.owner, 'Could not create a deep link for project without owner');
+
+  const url = new URL(`exp://expo.host/@${project.owner}/${project.slug}`);
```

Comment:
I think this should be `exp://exp.host` right, not `expo.host`.

Also, can you write up a test in [`tests/expo.test.ts`](https://github.com/expo/expo-github-action/blob/main/tests/expo.test.ts)? Something like:

```js
describe(projectDeepLink, () => {
  it('throws when owner is undefined', () => {
    expect(() => projectDeepLink({ name: 'fakename', slug: 'fakeslug' })).toThrow('without owner');
  });

  it('returns url with owner and slug', () => {
    expect(projectDeepLink({ name: 'fakename', slug: 'fakeslug', owner: 'fakeowner' })).toBe(
      'exp://exp.host/@fakeowner/fakeslug'
    );
  });

  it('returns url with owner, slug, and release channel', () => {
    expect(projectLink({ name: 'fakename', slug: 'fakeslug', owner: 'fakeowner' }, 'fakechannel')).toBe(
      'exp://exp.host/@fakeowner/fakeslug?release-channel=fakechannel'
    );
  });
});
```

😄